packages/tour-kit/src/components/tour-kit-frame.tsx

```diff
@@ -141,11 +155,24 @@ const TourKitFrame: React.FunctionComponent< Props > = ( { config } ) => {
 	}, [ config.options?.callbacks?.onMaximize, currentStepIndex ] );
 
 	useEffect( () => {
-		// first interactive element when step renders
+		/*
+		 * First interactive element when step renders
+		 */
 		setTimeout( () => initialFocusedElement?.focus() );
 	}, [ initialFocusedElement ] );
 
-	const classNames = classnames( 'tour-kit-frame', config.options?.className );
+	useEffect( () => {
+		/*
+		 * Fixes issue with Popper misplacing the instance on mount
+		 * See: https://stackoverflow.com/questions/65585859/react-popper-incorrect-position-on-mount
+		 */
+		update && update();
+		setTimeout( () => setTourReady( true ) );
+	}, [ update ] );
```

Comment:
Bug:

![Kapture 2021-12-06 at 15 45 30](https://user-images.githubusercontent.com/1705499/144856962-ac382680-5a50-4d41-80c8-be8e3de66edc.gif)

Fix without `tourReady` (brief flicker, not always visible):

![Kapture 2021-12-06 at 15 49 47](https://user-images.githubusercontent.com/1705499/144857661-ee30993c-ce5d-4dbc-b086-86422676fb51.gif)

Reply:
I made a test with the Welcome Tour and it seems that the "flickering" is still there: https://d.pr/v/24r3Xb (look when you arrive at the third point)

Reply:
Thanks! Yep, see it.

I think I've found a better fix 🤞 Take another look? :-)

Reply:
noticing this again while testing. looking

Reply:
Alright. Hopefully fixed now (albeit other dragons may lurk 😬 ). Give it a few more tests? :-)

This is what we should expect:

![Kapture 2021-12-08 at 14 11 48](https://user-images.githubusercontent.com/1705499/145206592-e2c1b3b5-4b8a-4f57-9fcd-3a063d8a398c.gif)

Reply:
Ok, now it seems to be working, well done!